src/KeyVault/KeyVault/Commands/Certificate/ImportAzureKeyVaultCertificate.cs

```diff
@@ -153,16 +173,22 @@ public override void ExecuteCmdlet()
                 ValidateParameters();
 
                 PSKeyVaultCertificate certBundle = null;
+                PSKeyVaultCertificatePolicy policy = null;
+                if (!string.IsNullOrEmpty(PolicyPath))
+                {
+                    policy = PSKeyVaultCertificatePolicy.FromJsonFile(PolicyPath);
```

Comment:
I prefer to a constructor from JsonFile to PSKeyVaultCertificatePolicy here.

Reply:
Since we have 
`internal static PSKeyVaultCertificatePolicy FromCertificatePolicy(CertificatePolicy certificatePolicy)` and 
`internal static PSKeyVaultCertificatePolicy FromTrack2CertificatePolicy(Track2Certificate.CertificatePolicy certificatePolicy)`

Maybe it is better to keep it as a transformer fucntion. Or it is better to change these three together to constructors.